source/guides/assets/content-delivery-network.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Lotus - Guides - Assets Content Delivery Network (CDN)
+---
+
+# Assets
+
+## Content Delivery Network (CDN)
+
+A Lotus application can serve assets from a [Content Delivery Network](https://en.wikipedia.org/wiki/Content_delivery_network) (CDN).
+This feature is useful in _production_ environment, where we want to speed up static assets serving.
+
+In order to take advantage of this feature, we need to specify CDN settings.
```

Comment:
@joneslee85 That isn't always the case. Eg. Heroku with CloudFront works transparently: https://discussion.heroku.com/t/using-aws-s3-with-asset-sync-cloudfront-or-just-cloudfront/313/5

Do you another case where syncing is still required? :+1: